packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -59,48 +59,42 @@ export function withResponsiveMode<TProps extends { responsiveMode?: ResponsiveM
 ): any {
   const resultClass = class WithResponsiveMode extends BaseDecorator<TProps, IWithResponsiveModeState> {
     private _events: EventGroup;
+    private _async: Async;
 
     constructor(props: TProps) {
       super(props);
       this._events = new EventGroup(this);
+      this._async = new Async(this);
```

Comment:
I believe you need a corresponding `this._async.dispose()` call in `componentWillUnmount` to avoid leaking memory.